CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 3.0.0a1 (Dec. 23, 2022)
+
+- update meshlab-pickedpoints
```

Comment:
In reality this release shouldn't change much for the consumer.  We're bumping meshlab-pickedpoints, but since we've skipped the version that didn't allow python3.7, there's really no functionality that's being added here.